### PR TITLE
B lorient802

### PR DESCRIPTION
**Pull request by MarcinKrawczuk**

### CODE REVIEW

File: LondonBritishLibrary/orient/BLorient802.xml [hunk 1]
```diff
@@ -55,71 +94,70 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                                 <extent>
                                     <measure unit="leaf">138</measure>
                                     <measure unit="blank">1</measure>
-                                    
-                                    
-                                    
-                                    
+
                                     <dimensions type="outer" unit="mm">
```
Comment (DariaElagina) [inline LondonBritishLibrary/orient/BLorient802.xml:98]: I think the catalogue provide inches, it should be changed to inches

File: LondonBritishLibrary/orient/BLorient802.xml [hunk 2]
```diff
@@ -55,71 +94,70 @@ schematypens="http://relaxng.org/ns/structure/1.0"
...
-                                <desc type="OwnershipNote">The note states that the manuscript belongs to the church of 
+                                <desc type="OwnershipNote">The note states that the manuscript belongs to the church of
                                     <placeName ref="INS0101MadhaneAlam"/>.</desc>
                                 <q xml:lang="gez">ፍትሐ፡ ነገሥት፡  ዘ<placeName ref="INS0101MadhaneAlam">ቅዱስ፡ መድኃኔ፡ ዓለም።</placeName>
-                           </q>
+                           </q></item>
+
+                           <item xml:id="a2">
+                           <locus target="#138v"/>
+                           <desc type="MixedNote">Lists of lands, in Amharic.</desc>
```
Comment (DariaElagina) [inline LondonBritishLibrary/orient/BLorient802.xml:136]: please, add an empty q with lang

File: LondonBritishLibrary/orient/BLorient802.xml [hunk 3]
```diff
@@ -55,71 +94,70 @@ schematypens="http://relaxng.org/ns/structure/1.0"
...
+
+
                     </physDesc>
-                    
+
                     <history>
                         <origin>
-                            
+
                             <origDate/>
```
Comment (DariaElagina) [inline LondonBritishLibrary/orient/BLorient802.xml:152]: the date can be provided based on the colophon

File: LondonBritishLibrary/orient/BLorient802.xml [hunk 4]
```diff
@@ -1,6 +1,6 @@
-<?xml version="1.0" encoding="UTF-8"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng" 
+<?xml version="1.0" encoding="UTF-8"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng"
 schematypens="http://relaxng.org/ns/structure/1.0"
-?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng" 
+?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/Schema/master/tei-betamesaheft.rng"
    schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI xmlns="http://www.tei-c.org/ns/1.0" xml:lang="en" xml:id="BLorient802" type="mss">
     <teiHeader xml:base="https://betamasaheft.eu/">
         <fileDesc>
```
Comment (DariaElagina) [inline LondonBritishLibrary/orient/BLorient802.xml:6]: add also your name as the editor

File: LondonBritishLibrary/orient/BLorient802.xml [hunk 5]
```diff
@@ -153,20 +191,22 @@ schematypens="http://relaxng.org/ns/structure/1.0"
             <textClass>
                 <keywords>
                     <term key="CanonLaw"/>
-                    
+
```
Comment (DariaElagina) [inline LondonBritishLibrary/orient/BLorient802.xml:194]: `<term key="ChristianLiterature"/>
<term key="CanonLaw"/>
<term key="Translation"/>
<term key="Paks2"/>`

**Review by DariaElagina — CHANGES_REQUESTED** [review]

**Review by thea-m — APPROVED** [review]
Thank you!

**Pull request merged**